packages/vercel/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@ai-sdk/vercel",
+  "version": "0.0.0",
```

Comment:
1.0.0-alpha.4

files under /.changeset might also need updates, and there are other link files (search for how other provider pkgs are referenced)